HVCAENx527App/Db/HVCAENx527gbl.db

```diff
@@ -168,3 +168,19 @@ record (stringin, "$(P):CHANLIST:DELIM")
 {
 	field(VAL, " ")
 }
+
+## record how many time event 11 (EVNTNO_T1) is posted
+record(calc, "$(P):EVNTO_T1:COUNT")
+{
+    field(SCAN, "Event")
+    field(EVNT, "11")
+    field(CALC, "A:=A+1;A")
+}
+
+## is EVNTO_T1 increasing
+record(calc, "$(P):EVNTO_T1:UPDATING")
+{
+    field(INPA, "$(P):EVNTO_T1:COUNT NPP")
+    field(CALC, "(B!=A?1:0);B:=A")
+    field(SCAN, "5 second")
+}
```

Comment:
Does anything use this? If so a quick comment would be useful, if I can't see that it's referenced anywhere I would be tempted to remove it as "dead code"...

Reply:
It was used by nagios. This addition was purely to allow us to easily monitor a potential hang in the update loop remotely and need to be in this commit to build a version for MUSR. Now I think we have proved the issue was a network disconnect that is now handled by the other additions this could be removed.